clinica/iotools/converters/adni_to_bids/adni_modalities/adni_t1.py

```diff
@@ -109,11 +108,8 @@ def compute_t1_paths(source_dir, csv_dir, subjs_list, conversion_dir):
     mri_quality = load_clinical_csv(csv_dir, "MRIQUALITY")
     mayo_mri_qc = load_clinical_csv(csv_dir, "MAYOADIRL_MRI_IMAGEQC_12_08_15")
 
-    # Keep only T1 scans
-    mayo_mri_qc = mayo_mri_qc[mayo_mri_qc.series_type == "T1"]
```

Comment:
@tharpm95 any reasons for removing this line from the T1 converter ?

Reply:
I can't recall any reason. Maybe I mistakenly left it removed during debugging to figure out how specific modalities are accessed to build the image list.